### PR TITLE
[WebServerBundle] Fix package type in composer.json

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | master

### CODE REVIEW

**Comment by chalasr** [comment]
Looks like there's no label for this bundle yet.

File: src/Symfony/Bundle/WebServerBundle/LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2004-2017 Fabien Potencier
+Copyright (c) 2016-2017 Fabien Potencier
```
Comment (fabpot) [inline src/Symfony/Bundle/WebServerBundle/LICENSE]: Should be reverted as some code was extracted from another place.

File: src/Symfony/Bundle/WebServerBundle/composer.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "symfony/web-server-bundle",
-    "type": "web-server-bundle",
+    "type": "symfony-bundle",
```
Comment (fabpot) [inline src/Symfony/Bundle/WebServerBundle/composer.json]: Good catch!

File: src/Symfony/Bundle/WebServerBundle/LICENSE (hunk 1 above)
Comment (chalasr) [inline src/Symfony/Bundle/WebServerBundle/LICENSE]: reverted

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**